src/YoutubeAtomOverlay.tsx

```diff
@@ -135,11 +135,11 @@ export const YoutubeAtomOverlay = ({
                 >
                     <SvgPlay />
                 </div>
-                {duration !== undefined && duration > 0 && (
+                {duration ? (
```

Comment:
Yes, it’s a refactor that I believe keeps thing clearer, but happy to drop it. Any thoughts @AshCorr, as you implemented the initial fix?

In any case it would not require a version bump as the result is identical.